### PR TITLE
Adding some functions to the atmoML model

### PR DESCRIPTION
**Pull request by SallyElHajjar**
I have added some functions for the atmoModel, similarly to the flood model. Still need to create the corresponding testing cases.

### CODE REVIEW

File: usl_models/usl_models/atmo_ml/model.py [hunk 1]
```diff
@@ -1,22 +1,133 @@
...
+            optimizer=tf.keras.optimizers.get(self._model_params["optimizer_config"]),
+            loss=tf.keras.losses.MeanSquaredError(),
+            metrics=[
+                tf.keras.metrics.MeanAbsoluteError(),
+                tf.keras.metrics.RootMeanSquaredError(),
+            ],
+        )
+        return model
+
+    def call(self, input: Input) -> tf.Tensor:
```
Comment (Katsutoshii) [inline usl_models/usl_models/atmo_ml/model.py:82]: Nit: add a short docstring here like we have in flood_ml/model.py.

**Review by Katsutoshii — APPROVED** [review]

File: usl_models/usl_models/atmo_ml/model.py (hunk 1 above)
Comment (Katsutoshii) [reply usl_models/usl_models/atmo_ml/model.py:82]: This is also required by flake8 checks

File: usl_models/usl_models/atmo_ml/model.py [hunk 2]
```diff
@@ -222,7 +333,7 @@ def __init__(
             name="wdir10_output_cnn",
         )
 
-    def call(self, inputs: AtmoInput) -> tf.Tensor:
+    def call(self, inputs: Input) -> tf.Tensor:
```
Comment (Katsutoshii) [inline usl_models/usl_models/atmo_ml/model.py:336]: Should be AtmoModel.Input

Comment (SallyElHajjar) [reply usl_models/usl_models/atmo_ml/model.py:336]: if  i have defined it as Input in the class Atmo, shouldn't keep it input here?

File: usl_models/usl_models/atmo_ml/model.py (hunk 1 above)
Comment (SallyElHajjar) [reply usl_models/usl_models/atmo_ml/model.py:86]: done.

File: usl_models/usl_models/atmo_ml/model.py (hunk 2 above)
Comment (SallyElHajjar) [reply usl_models/usl_models/atmo_ml/model.py]: done

File: usl_models/usl_models/atmo_ml/model.py [hunk 3]
```diff
@@ -1,22 +1,143 @@
...
+        """Prediction result dictionary."""
+
+        prediction: tf.Tensor
+        chunk_id: str | tf.Tensor
+
+    def __init__(
+        self,
+        params: AtmoModelParams | None = None,
+        spatial_dims: tuple[int, int] = (constants.MAP_HEIGHT, constants.MAP_WIDTH),
+        num_spatial_features: int = constants.num_spatial_features,
```
Comment (SallyElHajjar) [inline usl_models/usl_models/atmo_ml/model.py:34]: @Katsutoshii I have added these and some other changes, if you would like to check before merging

**Pull request merged**